authorization-tck/pom.xml

```diff
@@ -32,6 +32,9 @@
     <name>TCK: Authorization</name>
 
     <properties>
+        <tck.root>${project.build.directory}</tck.root>
+        <maven.settings.xml>${user.home}/.m2/settings.xml</maven.settings.xml>
+        <maven.executable>/opt/maven/bin/mvn</maven.executable>
```

Comment:
Can we utilise the MAVEN_HOME environment variable instead?

Reply:
Yes, if MAVEN_HOME is 3.9+

Reply:
Is it not just `${env.MAVEN_HOME}${file.separator}bin${file.separator}mvn` /  `${env.MAVEN_HOME}${file.separator}bin${file.separator}mvn.cmd` (Linux vs. Windows)? 
I don't think we need Maven 3.9 for that?

Reply:
done!